app/views/templates/project-panel.html.slim

```diff
@@ -21,7 +21,9 @@
         a ui-sref='help.gettingStarted({ organizationId: project.organizationId, projectId: project.id })'
           strong Getting Started
         '  guide to set it up.
-    .col-md-3.col-sm-4.col-xs-12 ng-if='stateChart'
+    .col-lg-4.col-md-5.col-sm-12 ng-if='project'
+      test-activity-chart ng-if='currentOrganization && project' type='projects' nb-days='15' organization='currentOrganization' project='project'
+    .col-lg-3.col-md-3.col-sm-12 ng-if='stateChart'
       .latest-state
         p.text-info Results of the last run
-        canvas.chart.chart-pie data='stateChart.data' labels='stateChart.labels' colours='stateChart.colors'
+        canvas.chart.chart-pie height='200px' data='stateChart.data' labels='stateChart.labels' colours='stateChart.colors'
```

Comment:
The browser is nice, but `height='200px'` is actually incorrect. The height attribute of a canvas is in pixels by definition, so it should be `height='200'`.